lib/node_modules/@stdlib/math/base/special/factorial2/package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
The contents of the `package.json` files have changed, take a look at the [snippet](https://github.com/stdlib-js/stdlib/blob/develop/tools/snippets/package.json). Fields that need to be updated: `author`, `contributors`, `repository`,  `homepage`, `bugs`, and `license`.